docs/source/model_design/vivarium_model_components/observers/index.rst

```diff
@@ -0,0 +1,386 @@
+.. _models_observers:
+
+======================
+Simulation Observers
+======================
+
+Simulation observers are unlike other types of Vivarium model components in that rather than adding to what is included in a Vivarium simulation, simulation observers simply record what is already modeled in the simulation. Simulation observers do just what their name implies: they "observe" specific quantities within the simulation and record those values so that the results of a simulation can be analyzed. It is important that we design simulation observers carefully so that we have the results that we need from each simulation run. This page discusses general guidelines in the design of simulation observers and how to structure observer documentation.
+
+.. contents::
+  :local:
+
+Observer background
+-------------------
+
+Observers are components in Vivarium and several standard observer packages are included in :code:`vivarium_public_health` (`see the documentation for these here <https://vivarium.readthedocs.io/projects/vivarium-public-health/en/latest/api_reference/results/index.html#module-vivarium_public_health.results>`_), but custom observers can also be written and included in vivarium simulations as well. As of August of 2025, the following table summarizes the standard observers available in :code:`vivarium_public_health` and the measures that they observe (termed observations).
+
+.. list-table:: Vivarium Public Health Observers
+  :header-rows: 1
+
+  * - Observer
+    - Observations 
+    - Note
+  * - MortalityObserver
+    - * Deaths
+      * Years of life lost (YLLs)
+    - By default, this counts cause-specific deaths and years of life lost over the full course of the simulation.
```

Comment:
Maybe it would be good to note that cause-specific deaths and ylls include deaths/ylls due to "other causes"?